docs/community/known-issues.md

```diff
@@ -29,3 +29,13 @@ If you are using [custom icons](/docs/customizations/icons), you might be unable
 
 ### Resolution
 For the time being, restart your docker container after adding / making any changes to your icons.
+
+## 🚨 403 status on most of applications
+If you are using [Nginx Proxy Manager](https://nginxproxymanager.com) for opening Homarr.
```

Comment:
```suggestion
If you are using [Nginx Proxy Manager](https://nginxproxymanager.com), some features like Torrent download clients or pings, may not work.
```

Reply:
I think need to be more specific about opening Homarr instance through proxy.
Its ok if we write like this?

```suggestion
If you are using [Nginx Proxy Manager](https://nginxproxymanager.com) to open Homarr instance, some features like Torrent download clients or pings, may not work.
```